games-engines/ponscripter/ponscripter-20111009.recipe

```diff
@@ -1,71 +1,61 @@
 SUMMARY="NScripter-like interpreter with proportional font and Unicode support"
-DESCRIPTION="A port of poular japanese Visual Novells engine with Unicode support"
-
+DESCRIPTION="Ponscripter is an NScripter-style novel-game interpreter with an \
+emphasis on supporting games in Western languages. The goal is to provide an \
+easy porting target for translation projects."
 HOMEPAGE="http://unclemion.com/onscripter/"
-SOURCE_URI="https://unclemion.com/dev/attachments/download/48/ponscripter-20111009-src.tar.bz2"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2001-2011 Ogapee
+	2006-2011 insani, Haeleth, Sonozaki et al"
+REVISION="5"
+SOURCE_URI="http://getdeb.megaglest.org/ubuntu/pool/games/p/ponscripter/ponscripter_$portVersion.orig.tar.bz2"
 CHECKSUM_SHA256="592a7cea2b89646c145eee095a66ef1c2b8a64c3d169b6b2b46da0829618ed3e"
-SOURCE_DIR="ponscripter-20111009-src"
+SOURCE_DIR="ponscripter-$portVersion-src"
+PATCHES="ponscripter_x86-$portVersion.patchset"
```

Comment:
might as well rename to ponscripter-$portVersion.patchset?

Reply:
Whoops, typos :)